docs/manual/scala/guide/cluster/SerializationAkkaJackson.md

```diff
@@ -0,0 +1,39 @@
+# Intro to Akka Jackson
+
+Following is a short summary of how to use the Akka Jackson serialiser in Lagom. For further details, refer to the [Serialization with Jackson](https://doc.akka.io/docs/akka/2.6/serialization-jackson.html) docs.
+
+## Using Akka Jackson Serialization in Lagom
+
+Setting up Akka Jackson serialization requires three steps.
+
+First, create a custom marker trait, for example `trait CommandSerializable`.
```

Comment:
Creating your own trait is not a lot of work, but I wonder if we should instead have `Jsonable` for Scala and have it configured out-of-the-box.

WDYT?

Reply:
I'm not sure. My first reaction was `no` but maybe it's not such a bad idea. I'm not sure where in the lagom codebase it would live, though.